Taarafo.Core/Services/Foundations/GroupPosts/IGroupPostService.cs

```diff
@@ -13,5 +14,6 @@ public interface IGroupPostService
     {
         ValueTask<GroupPost> AddGroupPostAsync(GroupPost groupPost);
         IQueryable<GroupPost> RetrieveAllGroupPosts();
+        ValueTask<GroupPost> RemoveGroupPostByIdAsync(Guid groupPostId);
```

Comment:
Should be `ValueTask<GroupPost> RemoveGroupPostByIdAsync(Guid groupId, Guid PostId);`